dataset/src/main/scala/frameless/TypedColumn.scala

```diff
@@ -278,6 +278,24 @@ sealed class TypedColumn[T, U](
     */
   def /(u: U)(implicit n: CatalystNumeric[U]): TypedColumn[T, Double] = self.untyped.divide(u).typed
 
+  /** Returns a descending ordering used in sorting
+    *
+    * apache/spark
+    */
+  def desc(implicit catalystOrdered: CatalystOrdered[U]): SortedTypedColumn[T, U] =
+    new SortedTypedColumn[T, U](withExpr {
+      SortOrder(expr, Descending)
```

Comment:
I might prefer this `(FramelessInternals.expr(untyped.desc))` slightly more since it uses the externally facing desc method rather than the "internal" expression `SortOrder`. What do you think?